ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml

```diff
@@ -78,3 +78,19 @@ tests:
   commands: ARTIFACT_DIR=/tmp/artifacts make unit
   container:
     from: src
+- artifact_dir: /tmp/artifacts
+  as: e2e-upgrade-v1.0
+  commands: ARTIFACT_DIR=/tmp/artifacts SOURCE=v1.0 make upgrade
+  secret: 
+    name: azure
```

Comment:
just a note for the future since I didn't know this:
ci-operator doesn't take the secret name directly from openshift, it picks up the mounted secret.